package_linter.py

```diff
@@ -437,6 +437,26 @@ def check_deprecated_practices(script):
     if script["name"] == "install" and "ynh_print_info" not in script["shlex"] and "ynh_script_progression" not in script["shlex"]:
         print_warning("Please add a few messages for the user, to explain what is going on (in friendly, not-too-technical terms) during the installation. You can use 'ynh_print_info' or 'ynh_script_progression' for this.")
 
+def check_source_common(script):
+    #test for standard execution
+    if script["name"] in ["install","upgrade","remove"]:
+        count_common=len(re.findall("^source _common.sh",script["raw"],flags=re.MULTILINE))
```

Comment:
`source ./_common.sh` is valid too in this case

Reply:
If you want, there are many syntax for calling the right file. I add you suggestion.

Reply:
done